pandas/tests/io/test_pickle.py

```diff
@@ -299,6 +300,19 @@ def test_pickle_v0_15_2():
     tm.assert_categorical_equal(cat, pd.read_pickle(pickle_path))
 
 
+def test_pickle_path_pathlib():
+    tm._skip_if_no_pathlib()
+    df = tm.makeDataFrame()
+    result = tm.round_trip_pathlib(df.to_pickle, pd.read_pickle)
+    tm.assert_frame_equal(df, result)
+
+
+def test_pickle_path_localpath():
+    tm._skip_if_no_localpath()
```

Comment:
I would move the skip routines to the actual test functions themselves (in util/testing)